### ISSUE TITLE
Source friendly name

### ISSUE THREAD

**Issue by lukemgriffith** [chocolatey/ChocolateyGUI#521]
Could we add the possibility of adding a friendly name to the source tab. My source names are generated and would be nice to have the ability to add a human friendly name to it.

**Comment by gep13** [comment]
@lukemgriffith this seems like a great idea to me!

**Comment by RichiCoder1** [comment]
This will probably need a sister issue in the main Chocolatey repo. We're currently just parroting what Choco is telling us is available.

**Comment by gep13** [comment]
@RichiCoder1 yeah, I had the same thought.  Originally I was thinking that we could augment the source information in Chocolatey GUI, but then I went back on that, as I think it would be a wrong direction.

**Comment by ferventcoder** [comment]
I like this, let's address it in an upcoming release.

**Comment by gep13** [comment]
@ferventcoder am I right in saying that this will require work in Chocolatey itself, before we can implement in Chocolatey GUI?  If so, we might have to bump this until that is available upstream.
